### PR TITLE
Mad Dank, a.k.a Android Fixes

### PR DESCRIPTION
**Pull request by alexhutman**
## Changes:
1. Fixed Android errors from Textbox component
2. Many style fixes/spruces 
3. Native pointer events
4. Mousepad works well now. Taps work, double taps work to select, you can hold off of a double tap, etc.

---

We probably want to rebase this one chief

### CODE REVIEW

File: components/Mousepad.tsx [hunk 1]
```diff
@@ -18,101 +20,117 @@ const browser_to_pictrl_clicks = new Map<
...
-            );
-            const mouseUpPacket = getMouseClickEventCommand(
-              browser_to_pictrl_clicks.get(button)!,
-              PICTRL_MOUSE_CLICK.UP,
-            );
-            conn.send(mouseDownPacket);
-            conn.send(mouseUpPacket);
-          }
+        // User double tapped
+        if (e.nativeEvent.timestamp - prev.lastMouseUp < MOUSE_DELAY_FOR_CLICK) {
```
Comment (mackalex) [inline components/Mousepad.tsx:72]: Really sick and, imo, the most intuitive way to implement a "double-tap-to-click."

File: components/Textbox.tsx [hunk 2]
```diff
@@ -1,39 +1,50 @@
...
+import { TextInput } from "react-native-gesture-handler";
 
 
-function getCodeForEvent(e: React.KeyboardEvent): number {
-  switch (e.key) {
+const SEND_ENTER_PROTOCOL_PACKET = getTextEventCommand('\n'.charCodeAt(0));
+
+function getCodeForEvent(e: NativeSyntheticEvent<TextInputKeyPressEventData>): number {
+  switch (e.nativeEvent.key) {
     case "Enter":
```
Comment (mackalex) [inline components/Textbox.tsx:12]: nit: not sure if we spent a lot of time looking into whether we can eliminate this case for the `"Enter"` key.

**Review by mackalex — APPROVED** [review]
Ack!

**Comment by alexhutman** [comment]
Whoops didn't see your approval, just please check those recent commits then I'll merge

**Comment by mackalex** [comment]
Ack, all good to go.

**Pull request merged**